numba/core/typing/typeof.py

```diff
@@ -265,3 +272,13 @@ def _typeof_nb_type(val, c):
         return types.NumberClass(val)
     else:
         return types.TypeRef(val)
+
+
+@typeof_impl.register(BitGenerator)
+def typeof_numpy_random_bitgen(val, c):
+    return types.NumPyRandomBitGeneratorType(val)
```

Comment:
seems like this is not covered by any tests

Reply:
I think it was added in [40e87e9](https://github.com/numba/numba/pull/8031/commits/40e87e903789460997195bb3d639ee48213fd13b)